include/odp/api/spec/classification.h

```diff
@@ -351,7 +351,11 @@ typedef struct odp_cls_cos_param {
 			odp_pktin_hash_proto_t hash_proto;
 		};
 	};
-	/** Pool associated with CoS */
+	/** Pool associated with CoS
+	 *
+	 * If the pktio does not support moving packets to another pool, this
+	 * parameter is ignored. @see odp_pktio_capability_t::cls_cos_pool
+	 */
```

Comment:
Pool parameter documentation could be improved:
- default value is POOL_INVALID
- when POOL_INVALID, pool configured to packet IO is used
- when !=POOL_INVALID, this pool is used
- new capa tells if this feature is supported. cos_create() returns failure if pool is set, but feature not supported. But we could leave it undefined (as usual) what happens if application acts against capa.